types/result.go

```diff
@@ -3,13 +3,13 @@ package types
 import (
 	"encoding/hex"
 	"encoding/json"
-	"math"
 	"strings"
 
 	abci "github.com/cometbft/cometbft/abci/types"
 	coretypes "github.com/cometbft/cometbft/rpc/core/types"
 	"github.com/cosmos/gogoproto/proto"
 
+	cmt "github.com/cometbft/cometbft/proto/tendermint/types"
```

Comment:
```suggestion
	cmtproto "github.com/cometbft/cometbft/proto/tendermint/types"
```

It's clearer which types it is then.

Reply:
gotcha